src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/CodeModel.java

```diff
@@ -122,4 +122,8 @@ public native final void insertRoxygenSkeleton() /*-{
       this.insertRoxygenSkeleton && this.insertRoxygenSkeleton();
    }-*/;
    
+   public native final int buildScopeTreeUpToRow(int row) /*-{
+      return this.$buildScopeTreeUpToRow && this.$buildScopeTreeUpToRow(row);
```

Comment:
Thanks, the test here is indeed wrong (fixed now). We do protect against this upstream (the scope tree is only built if we have a code model) but nonetheless this should be fixed.